xAPI-Data.md

```diff
@@ -467,11 +468,12 @@ The table below lists all properties of an Identified Group.
 
 ###### Requirements
 
-* A system consuming Statements MUST consider each Anonymous Group distinct even if it has an identical set of members.
-* Activity Providers SHOULD use an Identified Group when they wish to issue multiple statements, aggregate data 
-or store and retrieve documents relating to a group.
-* An Activity Provider MAY include a complete or partial list of Agents in the 'member' property of a given Anonymous or Identified Group.
-* An LRS returning a statement MAY return the list of group members in any order.
+* A Learning Record Consumer MUST consider each Anonymous Group distinct even if it has an identical set of members.
+* Learning Record Providers SHOULD use an Identified Group when they wish to issue multiple statements, 
+aggregate data or store and retrieve documents relating to a group.
+* A Learning Record Provider MAY include a complete or partial list of Agents in the 'member' property of a 
```

Comment:
I have found references to properties that are denoted with single quotes, double quotes, initcapped, and left naked. It would be good to maintain consistency across property references.